app/models/latest_blood_pressures_per_patient_per_quarter.rb

```diff
@@ -3,11 +3,18 @@ class LatestBloodPressuresPerPatientPerQuarter < ApplicationRecord
   include PatientReportable
 
   belongs_to :patient
+  has_one :materialized_latest_blood_pressure,
+    class_name: "LatestBloodPressuresPerPatient",
+    primary_key: :patient_id,
+    foreign_key: :patient_id
 
   def self.refresh
     Scenic.database.refresh_materialized_view(table_name, concurrently: true, cascade: false)
   end
 
   scope :with_hypertension, -> { where("medical_history_hypertension = ?", "yes") }
   scope :excluding_dead, -> { where.not(patient_status: :dead) }
+  scope :excluding_ltfu, ->(ltfu_as_of: Date.today) do
```

Comment:
Given the repetitions between this and per_month, would it make sense to extract these into a module?

Reply:
We could, I'll try to add this change if the scope of this PR doesn't end up being too big (which I'm afraid it's going to be). Will refactor in a separate PR otherwise 👍🏼 .

Reply:
Added in 2ec7c97.